packages/web/components/CurateSpecialties/CurateSpecialties.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { useModal } from '../../hooks';
+
+const CurateSpecialties = ({ data = [] }) => {
+	const { t } = useTranslation(['profile']);
+	const { openModal } = useModal();
+
+	const getChildrenCategory = (category) => {
+		const foundCategory = data.find((c) => c.parent_id === category);
+		return foundCategory ? foundCategory.term : 'Todas';
+	};
+
+	return (
+		<Container>
+			<h3>{t('profile:curateMySpecialties')}</h3>
+			<List>
+				{data.length > 0 ? (
```

Comment:
```suggestion
				{data.length ? (
```

A expressão `data.length` já é suficiente para o que você quer, não precisa do `> 0`